presidio-anonymizer/tests/test_presidio_app.py

```diff
@@ -0,0 +1,13 @@
+import json
+import os
+
+import app
+
+
+def test_anonymize():
```

Comment:
This is an integration test. which is great. Should consider using the Integration test framework which @SharonHart   contributed
It should not run in the context of regular unit tests

Reply:
I will do it. But will create a new ticket for it since I need to close some other things first :)

Reply:
Ticket 2664